### PR TITLE
Install oVirt SDK v4

### PR DESCRIPTION
**Pull request by nvgoldin**
Will be skipped for fc25, as it is not available there yet.

### CODE REVIEW

**Comment by nvgoldin** [comment]
if anyone wants to review this it the time ;), sort of need to get this fix applied today.

**Comment by eedri** [comment]
+1

**Comment by eedri** [comment]
Please verify with the manual OST Job to make sure we're not breaking anything

File: automation/check-patch.sh [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
-readonly INIT_FILE='automation/init-el73'
+readonly INIT_FILE='automation/lago-init'
 readonly TESTS_PATH='/tmp'
 readonly TIMEOUT="$((10 * 60))"
 
 function set_params() {
     ! [[ -c "/dev/kvm" ]] && mknod /dev/kvm c 10 232
     export LIBGUESTFS_BACKEND=direct
+    export LIBGUESTFS_DEBUG=1 LIBGUESTFS_TRACE=1
```
Comment (mykaul) [inline automation/check-patch.sh:10]: Unrelated change, but OK.

Comment (nvgoldin) [reply automation/check-patch.sh:10]: Right, forgot to separate it.

**Comment by nvgoldin** [comment]
@eedri - the script now has a different flow - as OST doesn't use it. 
http://jenkins.ovirt.org/job/lago-demo_master_github_check-patch-el7-x86_64/12/ - verified for fc24/centos7.

What it is lacking is downstream RHEL verification, for that I tested CSB manually, though it would be nice if we could make it automated someday.

File: automation/check-patch.sh (hunk 1 above)
Comment (nvgoldin) [reply automation/check-patch.sh:10]: fixed.

**Comment by nvgoldin** [comment]
ci merge please

**Pull request merged**